helpers/check_acme_routes.sh

```diff
@@ -148,37 +172,46 @@ function check_routes() {
 			fi
 
 			echo "$DNS_ERROR"
-			# Call the update function to update the route
-			update_annotation "$ROUTE_HOSTNAME" "$ROUTE_NAMESPACE"
-			notify_customer "$ROUTE_PROJECTNAME"
 
-			# Now once the main route is updated, it's time to get rid of exposers' routes
-			for j in $(oc get -n "$ROUTE_NAMESPACE" route|grep exposer|grep -E '(^|\s)'"$ROUTE_HOSTNAME"'($|\s)'|awk '{print $1";"$2}')
-			#for j in $(oc get -n $ROUTE_NAMESPACE route|grep exposer|awk '{print $1";"$2}')
-			do
-				ocroute=($(echo "$j" | tr ";" "\n"))
-				OCROUTE_NAME=${ocroute[0]}
-				if [[ $DRYRUN = true ]]; then
-					echo -e "DRYRUN oc delete -n $ROUTE_NAMESPACE route $OCROUTE_NAME"
-				else
-					oc delete -n "$ROUTE_NAMESPACE" route "$OCROUTE_NAME"
-				fi
-			done
+			if [[ "$NOTIFYONLY" = "true" ]]; then
+				echo $NOTIFYONLY
+				notify_customer "$ROUTE_PROJECTNAME"
+			else
+				echo $NOTIFYONLY
+				# Call the update function to update the route
+				update_annotation "$ROUTE_HOSTNAME" "$ROUTE_NAMESPACE"
+				notify_customer "$ROUTE_PROJECTNAME"
+
+				# Now once the main route is updated, it's time to get rid of exposers' routes
+				for j in $(oc get -n "$ROUTE_NAMESPACE" route|grep exposer|grep -E '(^|\s)'"$ROUTE_HOSTNAME"'($|\s)'|awk '{print $1";"$2}')
+				do
+					ocroute=($(echo "$j" | tr ";" "\n"))
+					OCROUTE_NAME=${ocroute[0]}
+					if [[ $DRYRUN = true ]]; then
+						echo -e "DRYRUN oc delete -n $ROUTE_NAMESPACE route $OCROUTE_NAME"
+					else
+						echo -e "\nDelete route $OCROUTE_NAME"
+						oc delete -n "$ROUTE_NAMESPACE" route "$OCROUTE_NAME"
+					fi
+				done
+			fi
 		fi
 		echo -e "\n"
+
+
 	done
 }
 
 # Function to update route's annotation (ie: update tls-amce, remove tls-acme-awaiting-* and set a new one for internal purpose)
 function update_annotation() {
 	echo "Update route's annotations"
-	OCOPTIONS=""
+	OCOPTIONS="--overwrite"
 	if [[ "$DRYRUN" = "true" ]]; then
```

Comment:
sometimes you use `[[ "$DRYRUN" = "true" ]]` or `[[ "$DRYRUN" == true ]]`, please use one way in the whole script.

Reply:
I have already realized this, and I have already fixed. Wondering if you are looking at the latest version I pushed yesterday.